src/payment/PaymentMethodSelect.jsx

```diff
@@ -8,7 +8,9 @@ import AcceptedCardLogos from './assets/accepted-card-logos.png';
 import { ApplePayButtonContainer } from './apple-pay';
 import { PayPalButton } from './paypal';
 
-function PaymentMethodSelect({ intl, loading }) {
+function PaymentMethodSelect({ intl, loading, isBasketProcessing }) {
+  const submissionDisabled = loading || isBasketProcessing;
```

Comment:
This reads a little odd to me, maybe submitDisabled would do?  Not a big deal though.